src/components/Gallery/Gallery.scss

```diff
@@ -143,13 +137,68 @@ $verticalGalleryMargin: 28px;
             }
         }
     }
+
+    &_mode_mobile {
+        &#{$block}_interactive {
+            #{$block} {
+                &__content {
+                    display: grid;
+                    grid-template: 'mobile-header' min-content 'body' auto 'mobile-footer' min-content;
+                }
+
+                &__body {
+                    flex: 1;
+                    min-height: 0;
+                    overflow: hidden;
+                }
+
+                &__header {
+                    position: static;
+                    opacity: 1;
+                    z-index: auto;
+                }
+            }
+        }
+
+        #{$block} {
+            &__body {
+                // Zoom support
+                overflow: hidden;
+
+                // File switching animation
+                transition: transform 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+            }
+
+            &__body-content {
+                overflow: hidden;
+                touch-action: none; // Prevent default touch behaviors during zoom
+
+                opacity: 1;
+                transform: translateX(0);
+                transition:
+                    opacity 0.25s ease-in-out,
+                    transform 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+            }
+
+            &__body-content_switching {
+                opacity: 0;
+                transform: translateX(20px);
+            }
+        }
+    }
 }
 
```

Comment:
The navigation is always visible on the full screen gallery now. Is it by design?

<img width="1379" alt="Screenshot 2025-06-15 at 09 18 30" src="https://github.com/user-attachments/assets/70cbb14f-dbe6-41e0-b358-8173a37147f4" />

Here is the previous version:

<img width="1381" alt="Screenshot 2025-06-15 at 09 21 29" src="https://github.com/user-attachments/assets/8647bd86-3213-4298-8467-07b87685228d" />
<img width="1377" alt="Screenshot 2025-06-15 at 09 21 17" src="https://github.com/user-attachments/assets/3250f3ab-67d4-405e-807d-29825ec19f36" />